src/test/config.ts

```diff
@@ -6,7 +6,25 @@ export const config = {
   SCENARIO_RETRY_LIMIT: 3,
   TestHeadlessBrowser: process.env.TEST_HEADLESS ? process.env.TEST_HEADLESS === 'true' : true,
   WaitForTimeout: 20000,
-
+  SUPER_ADMIN_CITIZEN_USER_LASTNAME: 'User',
+  superUser: {
+    email: 'superuser@test.com',
```

Comment:
Not sure if we want to update the email using the following format so that if the users failed to be deleted, it is obvious they are test users:
TEST_TEAMNAME_j2Xgie_USER_2j30fK

This is mentioned in the following doc:
https://tools.hmcts.net/confluence/display/SISM/Test-support+endpoint

Reply:
Good suggestion but currently hard to pull off, as discussed we'll look at adding in future